src/api/middleware/validators/offer.js

```diff
@@ -121,6 +139,11 @@ const get = useExpressValidators([
         .optional()
         .isInt({ min: 0, max: OfferService.MAX_OFFERS_PER_QUERY }).withMessage(ValidationReasons.INT)
         .toInt(),
+
+    query("showHidden")
+        .optional()
+        .isBoolean().withMessage(ValidationReasons.BOOLEAN)
+        .toBoolean(),
```

Comment:
This is to allow an admin to view them regardless in the future, maybe I should add that authorization logic right away? (Only show hidden if showHidden = true and isAdmin)

Reply:
Yes, I think that would be best, otherwise this might be forgotten later on and cause some issues if someone discovers this parameter somehow.
Furthermore, admins probably have access to the db anyway? Might be more annoying sure, but would make this pointless anyway :stuck_out_tongue:

Reply:
Nobody should have access to the DB, or need it in any case. But for some edge case, only the god has access to DB, and not admins, so I think this still makes sense